examples/webgl_multiple_views_multiview.html

```diff
@@ -0,0 +1,325 @@
+				scene.add( light );
+
+				// ground
+
+				var mesh = new THREE.Mesh( new THREE.PlaneBufferGeometry( 2000, 2000 ), new THREE.MeshPhongMaterial( { color: 0x999999, depthWrite: false } ) );
+				mesh.rotation.x = - Math.PI / 2;
+				scene.add( mesh );
+
+				var grid = new THREE.GridHelper( 200, 40, 0x000000, 0x000000 );
+				grid.material.opacity = 0.2;
+				grid.material.transparent = true;
+				scene.add( grid );
+
+				// balls
+				balls = new THREE.Group();
+				scene.add( balls );
+
+				var geometry = new THREE.IcosahedronBufferGeometry( radius, 2 );
+
+				var numObjects = urlParams.get('num') || 1000;
+
+				for ( var i = 0; i < numObjects; i ++ ) {
+
+					var object = new THREE.Mesh( geometry, new THREE.MeshLambertMaterial( { color: Math.random() * 0xffffff } ) );
+
+					object.position.x = Math.random() * 20 - 10;
+					object.position.y = Math.random() * 10 + 5;
+					object.position.z = Math.random() * 20 - 10;
+
+					object.userData.velocity = new THREE.Vector3();
+					object.userData.velocity.x = Math.random() * 0.01 - 0.005;
+					object.userData.velocity.y = Math.random() * 0.01 - 0.005;
+					object.userData.velocity.z = Math.random() * 0.01 - 0.005;
+
+					balls.add( object );
+
+				}
+
+
+				// model
+
+				var loader = new THREE.GLTFLoader();
+
+				loader.load( 'models/gltf/RobotExpressive/RobotExpressive.glb', function( gltf ) {
+
+					var model = gltf.scene;
+					var animations = gltf.animations;
+
+					scene.add( model );
+
+					var mixer = new THREE.AnimationMixer( model );
+					mixer.clipAction( animations[ 10 ] ).play();
+					mixers.push( mixer );
+
+					animate();
+
+				}, undefined, function( e ) {
+
+					console.error( e );
+
+				} );
+
+        var canvas = document.createElement( 'canvas' );
+				var context = canvas.getContext( 'webgl2', { antialias: false } );
+				renderer = new THREE.WebGLRenderer( { context: context, canvas: canvas, antialias: true, multiview: multiview } );
+				renderer.setPixelRatio( window.devicePixelRatio );
+				renderer.setSize( window.innerWidth, window.innerHeight );
+				renderer.gammaOutput = true;
+				renderer.gammaFactor = 2.2;
+				container.appendChild( renderer.domElement );
+
+				console.log(renderer.multiview);
+				//
+
+				function colorize( value ) {
+					return value ? '<b style="color:#3f3">true</b>' : '<b style="color:#f33">false</b>';
+				}
+
+				var info = document.createElement( 'div' );
+				info.style.position = 'absolute';
+				info.style.top = '10px';
+				info.style.width = '100%';
+				info.style.textAlign = 'center';
+				container.appendChild(info);
+
+				info.innerHTML = `<a href="http://threejs.org" target="_blank" rel="noopener">three.js</a> webgl - multiple views using multiview extension - by <a href="https://twitter.com/fernandojsg">fernandojsg</a><br/>`;
+				info.innerHTML += 'enabled: ' + colorize( renderer.multiview.isEnabled() ) + ` <a href="?num=${ numObjects }&${ multiview ? '' : 'enableMultiview' }">toggle</a>`
```

Comment:
Should this be `isAvailable`? It throws for me.

Reply:
Yeah, although after speaking with @mrdoob I'm going to remove this example as using array camera for multiple viewports is not recommended, at least in this use case because of the frustum culling issues.